From: Hanjun Guo <guohanjun@huawei.com>
Date: Thu, 24 Sep 2020 10:57:01 +0800
Subject: ACPI: event: Remove leftover ACPICA debug code
Patch-mainline: v5.10-rc1
Git-commit: 3ffa00e88df58f755b1a6ac20949ab09f92086af
References: jsc#SLE-16407

After commit (ff491a7334ac "netlink: change return-value logic of
netlink_broadcast()"), ACPI_DEBUG_PRINT() was removed from event.c,
so the ACPICA debug code is not used, remove it.

Signed-off-by: Hanjun Guo <guohanjun@huawei.com>
[ rjw: Subject edit ]
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/event.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/acpi/event.c
+++ b/drivers/acpi/event.c
@@ -19,9 +19,6 @@
 
 #include "internal.h"
 
-#define _COMPONENT		ACPI_SYSTEM_COMPONENT
-ACPI_MODULE_NAME("event");
-
 /* ACPI notifier chain */
 static BLOCKING_NOTIFIER_HEAD(acpi_chain_head);
 
